lib/streamlit/elements/time_widgets.py

```diff
@@ -399,14 +418,14 @@ def _time_input(
     def date_input(
         self,
         label: str,
-        value: DateValue = None,
+        value: DateValue | str | None | "builtins.ellipsis" = Ellipsis,
```

Comment:
Why do we need to add `str` here as type?

Reply:
It's a leftover when I used "..." instead of `Ellipsis` as default type, it's updated now 👍